src/sif/autocost.cc

```diff
@@ -7,7 +7,6 @@
 #include "baldr/nodeinfo.h"
 #include "midgard/constants.h"
 #include "midgard/util.h"
-#include "sif/dynamiccost.h"
```

Comment:
why are these diffs here. this branch isnt intended to change any of this. @PureW any clues as to how this is basically the inverse of your changes?

Reply:
i know rewriting git history can cause stuff like this but we havent ever rewritten history on master, so how can this be?